src/components/encounter/ohri-encounter-form.tsx

```diff
@@ -29,6 +29,9 @@ import { useEncounter } from '../../hooks/useEncounter';
 import { useInitialValues } from '../../hooks/useInitialValues';
 import { useEncounterRole } from '../../hooks/useEncounterRole';
 
+/**
+ * Todo pirupius - this needs to be refactored to pass the provider and location as props
+ */
```

Comment:
This will build on top of work being done 👉🏾  https://github.com/openmrs/openmrs-form-engine-lib/pull/30